scripts/build-wasm.sh

```diff
@@ -13,6 +13,9 @@ source "${HOME}/.nvm/nvm.sh"
 nvm use
 source "${HOME}/.cargo/env"
 
+echo "Setting rustup override for this project"
+rustup override set `cat rust-toolchain`
```

Comment:
Please, don't use backticks. Use `$()` as it is POSIX & can be nested (copy & pasted freely around).